### ISSUE TITLE
Check for update

### ISSUE THREAD

**Issue by SlimeDog** [RockinChaos/ItemJoin#232]
Spigot 1.14.4 and previous
ItemJoin 5.0.4-b293 and previous

Successful update check is reported in log
```
[06:58:31] [Server thread/INFO]: [ItemJoin] You are up to date!
```
even though
```
General:
  CheckforUpdates: false
```

**Once the following information is provided I will reply to you as quick as possible!**
1.) The version of the Resource.
2.) Version of the Spigot / Craftbukkit / Cauldron / ETC.
3.) What is the problem that you are having?
4.) Post all your YAML / Configuration files, please include a Pastebin.com link.
5.) Post any error codes found in your console window if it pertains to this issue, please include a Pastebin.com link.

**Comment by RockinChaos** [comment]
Fixed; http://ci.craftationgaming.com/job/ItemJoin/295/

**Comment by SlimeDog** [comment]
Confirmed fixed build 295. Thanks.

**Comment by RockinChaos** [comment]
This bug has been officially fixed in the release of ItemJoin v5.0.4.
You can grab it here; https://www.spigotmc.org/resources/itemjoin.12661/download?version=295178

Please read the changelog as the documentation has not been updated yet;
https://github.com/RockinChaos/ItemJoin/wiki/Recent-Changes
